application/deed/schemas/deed-schema-v0-1.xsd

```diff
@@ -49,7 +49,7 @@
 
     <xs:complexType name="signatureType">
         <xs:sequence>
-            <xs:element ref="dsig:Signature" minOccurs="0"></xs:element>
+            <xs:element name="signature" minOccurs="0"></xs:element>
```

Comment:
I think it's best to revert the ref="dsig:Signature" here, even though it's an outdated schema because this will fail schema validation on signing.